src/main/resources/view/DarkTheme.css

```
@@ -100,15 +99,15 @@
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: derive(#274156, 20%);
 }
 
 .list-cell:filled:odd {
-    -fx-background-color: #515658;
+    -fx-background-color: derive(#53687E, 10%);
 }
 
 .list-cell:filled:selected {
-    -fx-background-color: #424d5f;
+    -fx-background-color:derive(#7392b7, 10%);
```

I think there is a missing space.
```suggestion
    -fx-background-color: derive(#7392b7, 10%);
```